types/block.go

```diff
@@ -217,7 +219,25 @@ func (l *Layer) Blocks() []*Block {
 }
 
 func (l *Layer) Hash() []byte {
```

Comment:
shouldn't we return common.Hash?

Reply:
It doesn't matter Yuval already fixed it and we will conflict on that once I'll rebase/merge (and I'll pick Yuval's version).